Python stack tracing OD flow (part 1)

slgong-fb opened the pull request:
Summary: submodule update

Test Plan: CI

Differential Revision: D39176686

robieta reviewed (approved):
LGTM. I added extra CI since this update has hit speedbumps recently, so we should be careful.

malfet commented:
Looks like all failures are specific to CUDA-10.2 check, which makes me wonder if kineto is properly handling integration with this ancient CUDA version...

slgong-fb committed:
Python stack tracing OD flow (part 1) (#84362)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/84362

submodule update

Test Plan: CI

Reviewed By: robieta

Differential Revision: D39176686

fbshipit-source-id: ca58241ae07812d437b1132776fa2b118d1203ec

malfet commented:
@pytorchbot revert -m "Broke CUDA-10.2 tests, see https://hud.pytorch.org/pytorch/pytorch/commit/1f4f05e59c4cd72dfff9755629f7cc23f8df7abe" -c ignoredsignal
